src/snapshot.js

```diff
@@ -196,6 +196,10 @@ export function restorePlayerSnapshot(player, callback) {
       try {
         resume();
       } catch (e) {
+        player.ads.error({
+          errorType: videojs.Error.AdsResumeContentFailed
```

Comment:
should we pass it here as well?

Reply:
Good call! I updated this in the latest commit